L1Trigger/L1TMuonOverlapPhase1/src/Tools/CandidateSimMuonMatcher.cc

```diff
@@ -637,13 +701,27 @@ std::vector<MatchingResult> CandidateSimMuonMatcher::match(
       continue;  //no sense to do matching
     }
 
+    LogTrace("l1tOmtfEventPrint") << "CandidateSimMuonMatcher::match, tsof.globalPosition().eta() "
+                                  << tsof.globalPosition().eta();
+
+    //checking if the propagated track is inside the OMTF range, TODO - tune the range!!!!!!!!!!!!!!!!!
+    //eta 0.7 is the beginning of the MB2,
+    if ((std::abs(tsof.globalPosition().eta()) >= 0.7) && (std::abs(tsof.globalPosition().eta()) <= 1.3)) {
+      LogTrace("l1tOmtfEventPrint")
+          << "CandidateSimMuonMatcher::match trackingParticle IS in OMTF region, matching to the omtfCands";
```

Comment:
Previous numbers had this as `1.31`. Given the comment this looks like it is describing the same region? Should this be `1.3` or `1.31`? Could you please confirm and update the comment to reflect what that number represents?